### ISSUE TITLE
What is authnServices.py?

### ISSUE THREAD

**Issue by jpstroop** [loris-imageserver/loris#343]
Is https://github.com/loris-imageserver/loris/blob/development/authnServices.py just an example file? Seems odd to have it outside of the application if it's meant to do something. Based on what it's for I'd say it 1) it needs to be documented, 2) it needs to be moved either to the `loris` dir or the `misc` dir.

**Comment by bcail** [comment]
Yes, my understanding is that it's an example authentication service, and Loris redirects to a service like it if that's how the Auth service is configured. I agree it should be documented and I would move it to the misc directory. @azaroth42

**Comment by azaroth42** [comment]
Should go in misc/ as it's stand alone, and yes needs docs.

**Comment by azaroth42** [comment]
And feel free to assign to me.
